### ISSUE TITLE
decryption doesn't work on Android 8

### ISSUE THREAD

**Issue by takiainen** [TeamWin/android_device_oneplus_bacon#12]
Android Oreo roms cannot be decrypted with TWRP 3.2.1-0. This issues has been fixed for some phones (see https://twrp.me/leeco/leecolemax2.html, and https://github.com/TeamWin/Team-Win-Recovery-Project/issues/1168).

Any fix for bacon? thx

**Comment by vincez201** [comment]
If it helps, here's my recovery log:

[recovery.log](https://github.com/TeamWin/android_device_oneplus_bacon/files/2270928/recovery.log)

Also c.f. https://github.com/TeamWin/Team-Win-Recovery-Project/issues/1289

**Comment by mrplow** [comment]
This unofficial version has working decryption for Android 8.

https://forum.xda-developers.com/oneplus-one/development/recovery-twrp-3-0-3-k1-t3563692

**Comment by HugoPoi** [comment]
I confirm `TWRP 3.2.3-T3` from Kutep0v can decrypt data partition for bacon device and can apply OTA updates for Lineage 15.1 without problems.

**Comment by takiainen** [comment]
Oreo decryption seems to work now with TWRP 3.2.3-T3, but now there is issue with PIE decryption.

**Comment by ApolloLV** [comment]
[In this thread concerning mido](https://github.com/omnirom/android_bootable_recovery/issues/224), it is claimed that using 3.2.3-1 resolves the decryption issues. Can the same fix be applied to bacon as well and an official image be pushed?

**Comment by joergister** [comment]
yes it would be nice if  beacon would get a 3.2.3.-1 version too. This issue is a nasty one since due to it existing, I can't do any nandroid backups at the moment. Would be really helpful if this could get fixed for this device aswell.

**Comment by micxer** [comment]
I'd like to have a backup before upgrading to LOS 16, but it still doesn't work. Any chance to get this fixed soon?

**Comment by HugoPoi** [comment]
After updating to Lineage OS 16.0 I get stuck with the same issue again, but flashing recovery with `TWRP 3.2.3-K1` from Kutep0v solve this as indicated in the changelog of his version.
